documenting.rst

```diff
@@ -1803,10 +1803,9 @@ Ask on doc-sig, or better, make a PR on the `devguide
 I have a translation, but not on git, what should I do?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Just ask for help on the doc-sig mailing list and our python-fu, git-fu
-and bash-fu combined will help you create an appropriate repository. If
-you use a tool like transifex don’t worry.  Keeping them in sync is not
-that hard.
+You can ask for help on the doc-sig mailing list and our python-fu, git-fu
+and bash-fu combined will help you create an appropriate repository. Using
+a tool like transifex can help keeping them in sync.
```

Comment:
This significantly changes the meaning of the sentence; the point is that Transifex and GitHub *can* be kept in sync relatively easily, not that Transifex *helps* with keeping things (which things?) in sync.

Reply:
I concur with @zware, not trying to shame transifex but their sync tool is not that good, so it does *not* even really helps keeping things in sync.

(Also I personally don't find tx a great tool for Python documentation translation, it may be better for short strings, but for our paragraphs it does not really fit: I don't wand to advertise using it.)